framework-plugins/lisk-framework-forger-plugin/src/forger_plugin.ts

```diff
@@ -72,15 +73,22 @@ interface MissedBlocksByAddress {
 	[key: string]: number;
 }
 
+interface ForgerReceivedVotes {
+	[key: string]: Voters;
+}
+
 // eslint-disable-next-line
 const packageJSON = require('../package.json');
+const getBinaryAddress = (base64AddressStr: string) =>
+	Buffer.from(base64AddressStr, 'base64').toString('binary');
+const getAddressBuffer = (base64AddressStr: string) => Buffer.from(base64AddressStr, 'base64');
 
 export class ForgerPlugin extends BasePlugin {
 	private _forgerPluginDB!: KVStore;
 	private _server!: Server;
 	private _app!: Express;
 	private _channel!: BaseChannel;
-	private _forgersList!: ReadonlyArray<Forger>;
+	private _forgersList!: ds.BufferMap<boolean>;
```

Comment:
```suggestion
	private _forgersList!: dataStructures.BufferMap<boolean>;
```
prefer not to use abbreviation.